main.go

```diff
@@ -6,51 +6,56 @@ import (
 	"time"
 
 	"github.com/layer5io/gokit/logger"
+	"github.com/layer5io/gokit/utils"
 	"github.com/layer5io/meshery-kuma/api/grpc"
 	"github.com/layer5io/meshery-kuma/internal/config"
 	"github.com/layer5io/meshery-kuma/internal/tracing"
 	"github.com/layer5io/meshery-kuma/kuma"
 )
 
 var (
+	serviceName    = "kuma-adaptor"
```

Comment:
Let's keep it `adapter`

Reply:
It has to be `kuma-adaptor` because its an attribute for logger instance, not a constant.